### ISSUE TITLE
Create a schematic for the angular package.

### ISSUE THREAD

**Issue by doug0102** [cds-snc/gcds-components#54]
I think it would be useful to have an [angular schematic](https://angular.io/guide/schematics) so installation of the package would be a bit more streamlined. Currently you need to install the npm packages manually, update the app.module, then the global styles file. With a schematic this would all be taken care of for the user with just one command using the angular CLI.

**Comment by vid** [comment]
I'm in the middle of extracting a library from vue's version of schematic. Although there are some benefits to opionated "training wheels," it makes some things, especially things off the beaten path, much more complex. Are there any particular issues this would make better than using standalone tools?

**Comment by doug0102** [comment]
No not really, this was just a "nice to have" sort of thing. If it makes things needlessly difficult on your end then it might not be worth it.

**Comment by vid** [comment]
I don't think it would change anything for me, I'd mostly be concerned about porting in/out, and the extra overhead for onboarding people. It adds a lot of little fiddly scripts to maintain that are usually done differently, though I haven't seen its configure-on-install option before. OTOH I like to see any use of JSON-LD, and it looks like it's going in a components.js direction.

**Comment by ethanWallace** [comment]
Thanks @doug0102 for the suggestion, I will add it to my list of things to do!
